ci-operator/jobs/openshift/oc/openshift-oc-master-presubmits.yaml

```diff
@@ -117,6 +117,76 @@ presubmits:
         secret:
           secretName: sentry-dsn
     trigger: (?m)^/test( | .* )e2e-aws,?($|\s.*)
+  - agent: kubernetes
+    always_run: false
+    branches:
+    - master
+    context: ci/prow/e2e-aws-builds
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      ci-operator.openshift.io/prowgen-controlled: "true"
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-openshift-oc-master-e2e-aws-builds
+    rerun_command: /test e2e-aws-builds
+    run_if_changed: ^(pkg/.*build.*)|^(pkg/newapp)
```

Comment:
Only run if build-related and new-app packages are changed. We can add others in the future if we deem necessary.

Reply:
Needs to be `pkg/.*build.*` and `pkg/.*newapp.*`